gax/src/main/java/com/google/api/gax/retrying/BasicResultRetryAlgorithm.java

```diff
@@ -62,4 +82,18 @@ public TimedAttemptSettings createNextAttempt(
   public boolean shouldRetry(Throwable prevThrowable, ResponseT prevResponse) {
     return prevThrowable != null;
   }
+
+  /**
+   * Returns {@code true} if an exception was thrown ({@code prevThrowable != null}), {@code false}
+   * otherwise.
+   *
+   * @param prevThrowable exception thrown by the previous attempt ({@code null}, if none)
+   * @param prevResponse response returned by the previous attempt
+   */
+  @Override
+  public boolean shouldRetry(
```

Comment:
prev --> previous
Google style does not all abbreviations in variable names

Reply:
@elharo I've followed the existing style in the rest of this class (hierarchy), which all use `prev` instead of `previous`. What is your recommendation here?
* Apply Google style here, and leave all the other methods untouched?
* Also change the other methods to follow Google style?
* Keep with the existing style in this class?

Reply:
The Google Java style guide is explicit about this: existing violations do **NOT** justify new violations. Always follow the style guide.

Reply:
I've changed all new and existing parameter names in this class to comply with the style